nss-cohort-36/bangazon-client-virtual-magicians-1
# Creating initial file structure

# Description
Please check that the file structure is correct inside src.

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout cb-setup`
4. Please confirm that file structure is correct inside src

# Checklist:
